app/views/wiki/_tags.html.erb

```diff
@@ -1,4 +1,6 @@
 <% unless page.tag_list.empty? %>
-  <b><%= l :tags %>:</b>
-  <%= safe_join(page.tag_counts.collect{|t| render_tag_link(t, show_count: false, open_only: false) }, RedmineTags.settings[:issues_use_colors].to_i > 0 ? ' ' : ', ') %>
+  <div id="wiki-tags">
```

Comment:
I like the idea 👌 !

One nitpick, should this be an id? I believe a class would be a better fit since we are not referencing it anywhere.

Reply:
Yes, I questioned myself too.Now it's not referenced but in the future it may be necessary, so I'd stick with id